java-rmi/src/com/baeldung/rmi/RMIClientImpl.java

```diff
@@ -0,0 +1,23 @@
+package com.baeldung.rmi;
+
+import java.rmi.NotBoundException;
+import java.rmi.RemoteException;
+import java.rmi.registry.LocateRegistry;
+import java.rmi.registry.Registry;
+
+public class RMIClientImpl {
+
+	public static void main(String[] args) {
+		
+		try {
+			
+			Registry registry = LocateRegistry.getRegistry();
+			MessengerService server = (MessengerService) registry.lookup("MessengerService");
+			String responseMessage = server.sendMessage("Client Message");
```

Comment:
Is this a helpful class without some kind of output to the console? When I run this, there would just be no response, right?

Reply:
Just had written a client class. If only the integration test is to be included, this can be deleted.

Reply:
Code guidelines says not to write System.out.println(). Not sure that is just for articles or code in general. Thats why this has no output.

Reply:
I'll check with @pivovarit to see what he thinks. Like you said, might be immaterial here since the class is likely unnecessary.